### PR TITLE
[team_fantastic_0]: HelloWorld pr for users.

### PR DESCRIPTION
**Pull request by fanweiGit**
### What?
brief: 第一次pr
author: Fantastic

### What changes are included in this PR?
包含了一个团队目录、个人目录、个人学习内容。

### Are these changes tested?
测试代码见main.cpp，里面编写了一个helloworld程序。
image

### CODE REVIEW

File: stl/fantastic_team/fantastic/main.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿#include <iostream>
+
+int main()
+{
```
Comment (Light-City) [inline stl/fantastic_team/fantastic/main.cpp:4]: int main() {
}

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by fanweiGit** [comment]
thanks. I had changed the code style and directory structure.
image

File: stl/fantastic_team/README.md
Comment (Light-City) [inline stl/fantastic_team/README.md:1]: You should not copy ours but write your own

**Comment by Light-City** [comment]
[fantastic_team]-> [fantastic_team_0] 

[team name _ task number]

**Comment by Light-City** [comment]
reject

File: stl/fantastic_team/fantastic/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿#include <iostream>
+
+int main() {
+    std::cout << "hello world" << std::endl;
```
Comment (martinx) [inline stl/fantastic_team/fantastic/src/main.cpp:4]: Indent 2 spaces

**Comment by martinx** [comment]
pr title is wrong, should be

team_[team_name]_seq

```
[fantastic_team_0] => [team_fantastic_0]
```

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**